uc-crux-llvm/src/UCCrux/LLVM/Run/Simulate.hs

```diff
@@ -115,9 +126,96 @@ newtype CreateOverrideFn arch =
         IsSymInterface sym =>
         HasLLVMAnn sym =>
         sym ->
-        IO (PolymorphicLLVMOverride (Crux.Crux sym) sym arch)
+        IO (PolymorphicLLVMOverride arch (Crux.Crux sym) sym)
+    }
+
+newtype SymCreateOverrideFn sym arch =
```

Comment:
Could we get a note on what this is for?

Reply:
Done!